src/ec/packages/ec-component-accordion2/accordion2.story.js

```diff
@@ -3,32 +3,39 @@ import { storiesOf } from '@storybook/html';
 import { withKnobs, text } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
-
+import { buttonLabels, getExtraKnobs } from '@ecl-twig/story-utils';
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import demoData from './demo/data';
 
 import accordion2 from './ecl-accordion2.html.twig';
 import notes from './README.md';
 
+const PrepareAccordion2 = data => {
+  data.items.forEach((item, index) => {
+    const { content, toggle } = item;
+
+    item.toggle.label = text(
+      `toggle.label ${index}`,
```

Comment:
You saw what is the result of this?
`toggle.label 0`
do you recognise it as anything that would work in code? Clearly is not, but more than that you would still miss the items array where these properties have to be set, the label is supposed to be the "code to access it", not sure i'm able to convince you about this, but this should be:
`items[${index}].toggle.label`